addon/authenticators/oidc.js

```diff
@@ -124,6 +125,8 @@ export default BaseAuthenticator.extend({
         retryCount < amountOfRetries - 1
       ) {
         later(this, this._refresh, refresh_token, retryCount + 1, retryTimeout);
+        // Return old token as we couldn't refresh yet
+        return this._currentToken;
```

Comment:
Would it not make more sense to return the current `sessionData` instead of adding more state to this addon?